ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/spec/sum_with_out_max_min_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rspec/autorun'
+require '../sum_with_out_max_min.rb'
+
+describe SumWithOutMaxMin do
+
+    describe ".sort_array_of_intergers" do
+        it "should return a sorted array from smallest to largest value" do
```

Comment:
Another possible description could be `it "should return an array sorted on ascending order"`